prefect_databricks/flows.py

```diff
@@ -447,7 +627,7 @@ def jobs_runs_wait_for_completion_flow():
 
         jobs_runs_metadata = await jobs_runs_metadata_future.result()
         jobs_status = _update_and_log_state_changes(
-            jobs_status, jobs_runs_metadata, logger, "Job"
+            jobs_status, jobs_runs_metadata, logger, b"Job"
```

Comment:
I don't think this needs to be a `bytes` string
```suggestion
            jobs_status, jobs_runs_metadata, logger, "Job"
```